packages/context/src/context.ts

```diff
@@ -23,7 +27,7 @@ export class Context {
    */
   readonly name: string;
   protected readonly registry: Map<string, Binding> = new Map();
-  protected _parent?: Context;
+  protected _parent?: Context| string;
```

Comment:
I believe `_parent` must be always a Context instance. What would be the point of having the parent as a string name only?

Can you revert changes in this file please?

Reply:
@bajtos I made this change because `sdocs` throws error for it when generate the apidoc. 
Seems I misunderstood [line 36](https://github.com/strongloop/loopback-next/pull/1327/files#diff-39c875eaefcb5bd189d86d709238cb64R36), I thought it implies the property declaration on line 30 misses a union type, but `Context | string` is just a way to deal with the omitted parameter...